pkg/storegateway/bucket_test.go

```diff
@@ -2747,7 +2747,7 @@ func TestLabelNames_Cancelled(t *testing.T) {
 
 	_, err := store.LabelNames(ctx, req)
 	assert.Error(t, err)
-	s, ok := status.FromError(err)
+	s, ok := grpcutil.ErrorToStatus(err)
```

Comment:
[nit]
Another possibility would be:

```suggestion
	code := grpcutil.ErrorToStatusCode(err)
```

and then to check 
```
require.Equal(t, codes.Canceled, code)
```